docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -52,3 +70,20 @@ function yukonca_glider_preprocess_node(&$variables) {
     }
   }
 }
+
+/**
+ * Implements template_proprocess_page().
+ */
+function yukonca_glider_preprocess_page(&$variables) {
+  if (!empty($variables['node'])) {
+    $bundle = $variables['node']->bundle();
+    if ($bundle == 'landing_page' || $bundle == 'landing_page_level_2') {
```

Comment:
Can we somehow make this logic more universal so it would work with other bundles?

Reply:
I've set the code to apply to any node that has field_icon_dark set.